lib/event-listeners/foods-event-listeners.js

```diff
@@ -23,4 +23,12 @@ export function foodListeners() {
     let q = $('.filter-form [name=filter-form-value]').val()
     search(q)
   })
+
+  $('#foods-table').on("focusout", ".food-table-cell", () => {
```

Comment:
This is a great way to locate a cell, and enable content editable, however, I *think*-? anytime you click on ANYTHING on the page, all of the cells, not clicked on, will update/send patch requests, because they are assigned a state of 'focusout'.  You can open up your inspector, and watch the requests that are being made - see if that's what's happening.

Reply:
The second argument, ".food-table-cell" scopes this event handler down to only when nodes with the HTML class of "food-table-cell" are out of focus (from previously being focused and, possibly, edited).

Reply:
Oh yeah, I saw the second argument, but all of the food cells are labeled as such, with no identifiers, so I was concerned - honestly though, it was more about scalability than anything haha, not really an issue with quantified self :)